linera-service/tests/linera_net_tests.rs

```diff
@@ -2085,19 +2112,22 @@ async fn test_wasm_end_to_end_allowances_fungible(config: impl LineraNetConfig)
     app2.assert_balances(expected_balances).await;
     app3.assert_balances(expected_balances).await;
 
-    // Approving a transfer
+    // Approving a transfer.
     app1.approve(&owner1, &owner2, Amount::from_tokens(93))
         .await;
 
     app1.assert_allowance(&owner1, &owner2, Amount::from_tokens(93))
         .await;
 
-    // Call process inbox in order to synchronize from validators
-    node_service2.process_inbox(&chain2).await?;
-    app2.assert_allowance(&owner1, &owner2, Amount::from_tokens(93))
-        .await;
+    // Call process inbox in order to synchronize from validators.
```

Comment:
We don't process the inbox anymore and `get_allowance` doesn't mutate anything. How does it work now?